andela/valhalla-cfh
# Ch grunt to gulp 157801241

#### What does this PR do?
Change from grunt task runner to gulp task runner

#### Description of Task to be completed?
allow gulp to automate the following tasks:
1. transpile .scss files to .css
2. watch the necessary files for changes during development
3. synchronize code and browser using nodemon and browersync
4. install dependencies using bower

#### How should this be manually tested?
1. Clone/ pull the repository
2. run `npm install`
3. run `npm run start` to start application. Application should be live at `localhost:3000`
4. run `npm run test` to run test

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#157801241

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
